app/src/main/res/layout/dashboard_list_item_tools.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <data>
+        <import type="androidx.databinding.ObservableField" />
+        <import type="androidx.lifecycle.LiveData" />
+        <import type="org.cru.godtools.base.ui.util.ModelUtils" />
+
+        <variable name="callbacks" type="ObservableField&lt;org.cru.godtools.ui.tools.ToolsAdapterCallbacks&gt;" />
+        <variable name="tool" type="org.cru.godtools.model.Tool" />
+        <variable name="banner" type="LiveData&lt;org.cru.godtools.model.Attachment&gt;" />
+        <variable name="downloadProgress" type="LiveData&lt;org.cru.godtools.download.manager.DownloadProgress&gt;" />
+        <variable name="primaryTranslation" type="LiveData&lt;org.cru.godtools.model.Translation&gt;" />
+        <variable name="parallelTranslation" type="LiveData&lt;org.cru.godtools.model.Translation&gt;" />
+        <variable name="parallelLanguage" type="LiveData&lt;org.cru.godtools.model.Language&gt;" />
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="160dp"
+        android:padding="5dp">
```

Comment:
my guess is you really don't need this containing `FrameLayout`

Reply:
this padding is probably part of what is adding unnecessary whitespace to the UI